pkg/helpers/helpers.go

```diff
@@ -167,3 +167,14 @@ func Retry(attempts int, sleepDuration time.Duration, f func() (error, bool)) er
 	}
 	return fmt.Errorf("giving up after %v attempts", attempts)
 }
+
+// MatchStringsRegex returns a regex matching every string in stringSlice
+func MatchStringsRegex(stringSlice []string) *regexp.Regexp {
```

Comment:
1. Please unit-test it.
2. Let's move it to acceptance package (for now).
3. is `fmt.Println(a)` still needed? If so, let's add a DEBUG log instead.
4. escapedMessages -> escapedParts
5. stringSlice -> parts
6. MatchStringsRegex -> MatchAllStrings?
7. Order matter currently, right? Let's be explicit about this MatchAllStrings -> MatchAllStringsInOrder or smth

Reply:
Right, the order matters here. Added unit tests and changed some names.